scene/gui/rich_text_label.compat.inc

```diff
@@ -46,11 +54,18 @@ bool RichTextLabel::_remove_paragraph_bind_compat_91098(int p_paragraph) {
 	return remove_paragraph(p_paragraph, false);
 }
 
+void RichTextLabel::_set_table_column_expand_bind_compat_101482(int p_column, bool p_expand, int p_ratio) {
+	set_table_column_expand(p_column, p_expand, p_ratio, true);
+}
+
```

Comment:
I'd move this next to the `_bind_compat_79053` version as they're related (same with the `bind_compatibility_method`).